addons/auth_ldap/models/res_company_ldap.py

```diff
@@ -74,6 +74,9 @@ def _connect(self, conf):
         uri = 'ldap://%s:%d' % (conf['ldap_server'], conf['ldap_server_port'])
 
         connection = ldap.initialize(uri)
+        ldap_chase_ref_disabled = self.env['ir.config_parameter'].sudo().get_param('disable_ldap_chase_ref')
```

Comment:
could we use a proper namespace for the parameter, to follow our usual convention?

```suggestion
        ldap_chase_ref_disabled = self.env['ir.config_parameter'].sudo().get_param('auth_ldap.disable_chase_ref')
```

Also note that you're directly using the textual value of the parameter, so when set to "False" it will still mean that the option is ON. That's ok for me, but worth noting. That was the reason for the `str2bool()` in my example https://github.com/odoo/odoo/pull/29188#issuecomment-782003289

Reply:
@odony that's true my bad, I committed your suggestion, thank you for your feedback :pray:

Reply:
Of note though: we don't generally use `str2bool` / text content for boolean parameters. e.g. the bridge between config parameters and settings works on a set/unset basis.

Reply:
(furthermore even if it were desirable `odoo.tools.convert.str2bool` is the wrong one).

Reply:
@xmo-odoo thank you for your feedback :pray: , I changed `str2bool` to `odoo.tools.misc.str2bool`